tests/e2e/flowtest/flowtest_helpers.py

```diff
@@ -0,0 +1,223 @@
+import logging
+
+from ocs_ci.ocs import node, platform_nodes, defaults, exceptions
+from ocs_ci.ocs.node import wait_for_nodes_status
+from ocs_ci.ocs.resources import pod as pod_helpers
+from ocs_ci.ocs.resources.pod import wait_for_storage_pods
+from ocs_ci.utility.utils import TimeoutSampler
+from tests.manage.mcg.helpers import s3_delete_object, s3_put_object, s3_get_object
+from tests.sanity_helpers import Sanity
+from ocs_ci.framework.testlib import ignore_leftovers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pvc import delete_pvcs
+from tests import helpers
+from tests.manage.z_cluster.pvc_ops import create_pvcs
+
+logger = logging.getLogger(__name__)
+
+
+class FlowOperations:
+    def __init__(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+
+    def node_operations_entry_criteria(self, node_type, number_of_nodes, network_fail_time=None):
+        """
+        Entry criteria function for node related operations
+        Args:
+            node_type (str): Tyoe of node
+            number_of_nodes (int): Number of nodes
+            network_fail_time (int): Total time to fail the network in a node
+        Returns:
+            tuple: containing the params used in Node operations
+
+        """
+        logger.info("Node operations entry: Verifying cluster health")
```

Comment:
The lines 38 to 43 are common to both node_operations_entry_criteria() and node_operations_exit_criteria. Can they be under one generic function?..

Reply:
Created a generic function called validate cluster